LICENSE

```diff
@@ -1,661 +1,14 @@
-                    GNU AFFERO GENERAL PUBLIC LICENSE
-                       Version 3, 19 November 2007
+BSD 3-Clause License
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
- Everyone is permitted to copy and distribute verbatim copies
- of this license document, but changing it is not allowed.
+Copyright (c) 2020, Robert Gieseke, Zebedee Nicholls, Jared Lewis, Sven Willner, Matthias Mengel
```

Comment:
```suggestion
Copyright (C), Pymagicc developers and contributors
```
Maybe add a "see setup.py" for the full list?

Reply:
Fixed by ec01a5